project-fortis-pipeline/docs/user/admin.md

```diff
@@ -0,0 +1,88 @@
+| Header Logo Banner  | Image path of banner.  |
+| Supported Languages | A comma delimited string with languages formatted as language codes like `en` or `ar`. The admin site can translate to any of the supported languages.
+| Default Language    | The admin site will be displayed in this language. |
+| Feature Service     | The feature service connection string formatted as `postgres://frontend:{your_password_here}@127.0.0.1/features`. More documentation on the feature service [here](https://github.com/CatalystCode/featureService).  |
+| Translation Services Token | Needed for real-time text translation. Get a token [here](https://azure.microsoft.com/en-us/services/cognitive-services/translator-text-api).  |
+| Cognitive Speech Services Token | Get a token [here](https://azure.microsoft.com/en-us/services/cognitive-services/custom-speech-service/).  |
+| Cognitive Vision Services Token | Get a token [here](https://azure.microsoft.com/en-us/services/cognitive-services/custom-vision-service/)  |
+| Cognitive Text Services Token | Used for sentiment analysis. Get a token [here](https://azure.microsoft.com/en-us/services/cognitive-services/text-analytics/)   |
+
+## Watchlist
+
+Manage keywords you would like to monitor. Keywords belong to different categories, which you will define in `watchlist` settings.
+
+| Column Name         | Description   |
+| ------------------- | ------------- |
+| Category            | Categories are used to classify different keywords into logical groups. |
+| name                | Keyword in the site's default language. |
+| name_{language}     | Keyword translated to the site's supported language. |
+
+### Adding Keywords with Excel then Translating them to a Supported Language
+
+To add keywords quickly, you can copy keyword categories and keywords from `excel`, then paste them into the corresponding columns of the watchlist table. To get keyword translations in a supported languages, check the rows you would like translations for and then press the button `translate selection(s)`.
+
+For example, in Excel you may have:
+
+| Category            | Name          |
+| ------------------- |:-------------:|
+| armed conflict      | ammo          |
+| armed conflict      | gun           |
+| health              | medicine      |
+
+## Monitored Places / Geofence
+
+Specify coordinates and the zoom level of a location you would like to monitor.
+
+| Value               | Description          |
+| ------------------- | ------------- |
+| Target Bbox         | Uses WOF. Provide a comma delimited string `maxY,minX,minY,maxX` |
+| Zoom Level          | Specify as an `Integer`. |
+
+## Event Import
+
+Still needs to be implemented.
+
+## Trusted Sources
+
+Manage trusted sources like `twitter`, `facebook`, etc.
+
+| Column Name         | Description   |
+| ------------------- | ------------- |
+| Pipeline Key        | Key associated with a pipeline. |
```

Comment:
What specifically should this be? Given an example please, e.g. for a Twitter pipeline.